SQL/New_patches/2021-07-28_diagnosis_evolution.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE `diagnosis_evolution` (
+  `DxEvolutionID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) DEFAULT NULL,
+  `ProjectID` int(10) unsigned DEFAULT NULL,
```

Comment:
```suggestion
  `ProjectID` int(10) unsigned NOT NULL,
```

should this be not null because an evolution needs to belong to a project specific evolution?